spell-checker/main.js

```diff
@@ -0,0 +1,39 @@
+function spellChecker(userWords, correctWords)  {
+  var spellingObject = {};
+  var incorrect = [];
+  for (var i = 0; i < userWords.length; i++)  {
+    if (userWords[i] !== correctWords[i]) {
+      incorrect.push(userWords[i]);
+    }
+  }
+  spellingObject['incorrect'] = incorrect;
+  spellingObject['correct'] = correctWords;
```

Comment:
This shouldn't be the list of all correct words, it should be the list of words that were spelled correctly by the user 👍